pkg/agent/openflow/client.go

```diff
@@ -612,6 +656,8 @@ func (c *client) InstallGatewayFlows() error {
 	if gatewayConfig.IPv4 != nil {
 		gatewayIPs = append(gatewayIPs, gatewayConfig.IPv4)
 		flows = append(flows, c.gatewayARPSpoofGuardFlow(gatewayConfig.IPv4, gatewayConfig.MAC, cookie.Default))
+		// This is for Service virtual IPv4 ARP resolution when the source IP of the ARP request is not Antrea gateway's.
```

Comment:
I do not get the flow is for what exact case from the comments. Could you add more information here?

Reply:
This flow is used to allow the response Service packet whose Endpoint is on host network to get ARP resolution. The source IP of the ARP is host IP, not Antrea gateway's IP.